src/components/Nav/Aside.js

```diff
@@ -0,0 +1,276 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import Service from './AsideService';
+import { AiOutlineArrowRight, AiFillGithub } from 'react-icons/ai';
+import { IoIosArrowForward, IoIosArrowDown } from 'react-icons/io';
+import { BsFillJournalBookmarkFill, BsHouse } from 'react-icons/bs';
+import HostModal from './HostModal';
+
+const Aside = ({ aside, toggle }) => {
+  const token = localStorage.getItem('access_token');
+  const [service, setService] = useState(false);
+  const [host, setHost] = useState(false);
+
+  const serviceToggle = () => {
+    setService(!service);
+  };
+  const hostToggle = () => {
+    setHost(!host);
+  };
+  function test() {
+    localStorage.clear();
+    navigate('/main');
+  }
+  const navigate = useNavigate();
+  const Login = () => {
+    navigate('/login');
+  };
+  const HostLogin = () => {
+    navigate('/HostLogin');
+  };
+  return (
+    <AsideBox aside={aside}>
+      <AsideBoxfirst>
+        <GuestBox>
+          {token ? (
+            <div>
+              <LoginSignIn>노영완님!!!</LoginSignIn>
+              <LoginSignIn>환영합니다!!!</LoginSignIn>
+            </div>
+          ) : (
+            <div>
+              <LoginSignIn onClick={Login}>로그인</LoginSignIn>
+              <LoginSignIn>회원가입</LoginSignIn>
+            </div>
+          )}
+          <CloseBox>
+            <AiOutlineArrowRight onClick={toggle} />
+          </CloseBox>
+        </GuestBox>
+        <HostBox onClick={hostToggle}>
+          <Host>호스트로</Host>
+          <Host>등록하기</Host>
+        </HostBox>
+        {host === true ? <HostModal hostToggle={hostToggle} /> : null}
```

Comment:
```suggestion
        {host &&<HostModal hostToggle={hostToggle} />}
```
굳이 삼항연산자로 true, false 케이스 모두 처리하여 null을 반환할 필요없이 이 부분은 true 케이스만 처리하면 되기 때문에 조건부 렌더링 (`&&` 연산자)을 활용하면 불필요한 코드를 없애 더욱 가독성에 좋아집니다. 
- 참고 : 공식문서 조건부 렌더링([https://ko.reactjs.org/docs/conditional-rendering.html](https://ko.reactjs.org/docs/conditional-rendering.html))
```jsx
true ? foo : null => true && foo // true인 경우에 foo 요소를 렌더
```

Reply:
수정했습니다. 이제부터는 불필요한 코드는 줄이겠습니다!!